test/fixture.test.js

```diff
@@ -0,0 +1,104 @@
+
+import { defineCE, expect, waitUntil } from '@open-wc/testing';
+import { html, LitElement, nothing } from 'lit';
+import { fixture } from '../src/index.js';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+
+const resolves = new Map();
+
+const slowElem = defineCE(
+	class extends LitElement {
+		static get properties() {
+			return { nested: { type: Boolean } };
+		}
+		constructor() {
+			super();
+			this.nested = false;
+			this.finished = false;
+			this.promise = new Promise((resolve) => resolves.set(this.id, resolve));
+			this.promise.then(() => this.finished = true);
+		}
+		render() {
+			return this.nested ? html`${unsafeHTML(`<${slowElem} id="nested"></${slowElem}>`)}` : nothing;
+		}
+		async getUpdateComplete() {
+			await super.getUpdateComplete;
+			return this.promise;
+		}
+	}
+);
+
+describe('fixture', () => {
+
+	describe('waitForElem', () => {
+
+		afterEach(() => {
+			resolves.clear();
+		});
+
+		it('should wait for element at fixture root', async() => {
+			const finishedPromise = fixture(`<${slowElem} id="elem1"></${slowElem}>`)
+				.then((elem) => elem.finished);
+			await waitUntil(() => resolves.has('elem1'));
+			const timeout = setTimeout(() => resolves.get('elem1')(), 50);
```

Comment:
I hate having a real delay in these, but this was the only way I could get these tests to actually fail with our custom `fixture` removed. It's really difficult to test if something is waiting long enough when your test has to also wait that long to prove that it worked!